deploy/eck-beats/examples/auditbeat_hosts.yaml

```diff
@@ -0,0 +1,113 @@
+name: auditbeat
+type: auditbeat
+version: 8.2.3
+spec:
+  elasticsearchRef:
+    name: elasticsearch
+  kibanaRef:
+    name: kibana
+  config:
+    # Since filebeat is used in the default values, this needs to be removed with an empty list. 
```

Comment:
This is mentioned in several places, does it not feel a bit odd that the user must explicitly disable a `Beat` to deploy another one?

Reply:
Over the course of the previous Helm Chart PRs, we've tried to maintain that installing with the default values gives some sort of useful "default" installation.  Though just recently, in the agent Helm Charts PR, the `config` section in the default values was removed, so maybe this makes sense here as well, but the default will not be useful, but will make the user experience for overrides much better.  Thoughts?

Reply:
Sorry, for the very late answer, I need to have a look at what is proposed for Agent. Defaulting to `Filebeat` feels a bit "opinionated" to me. It means we assume that the "most useful" default is Filebeat, not Metricbeat, and I'm not sure I get the reason why.

Reply:
I see why defaulting to `Filebeat` may not make the most sense here.  I'll update this to use`Metricbeat` by default.

Reply:
actually @barkbay, I remember why I defaulted to filebeat once I went back and looked at this.  By defaulting to filebeat, we add the minimum amount to the default values and still have a useful "default" Beat install.  If we move this to metricbeat, many more default values would be needed to be added to make a useful default Beat install.  Let me know your thoughts here.